src/services/dashboards/widgets/budget-status/BudgetStatus.vue

```diff
@@ -206,6 +206,8 @@ defineExpose<WidgetExpose<Data>>({
         }
         .legend-wrapper {
             @apply text-gray-600;
+            position: absolute;
```

Comment:
It is hard to find `position: relative` on parent element.
Is it ok ?

Reply:
I missed it! Thanks!